src/Model/GalleryInterface.php

```diff
@@ -15,6 +15,9 @@
 
 use Doctrine\Common\Collections\Collection;
 
+/**
+ * @phpstan-template T of GalleryItemInterface
```

Comment:
This template is useless if it's not use anywhere else.

Either it should be removed or be used in the phpdoc of 
```
public function setGalleryItems(Collection $galleryItems): void;
    public function getGalleryItems(): Collection;
    public function addGalleryItem(GalleryItemInterface $galleryItem): void;
    public function removeGalleryItem(GalleryItemInterface $galleryItem): void;
```

And if we remove it it's not needed in the BaseGallery, Gallery, ...